posthog/hogql/hogql.py

```diff
@@ -311,43 +322,67 @@ def translate_ast(node: ast.AST, stack: List[ast.AST], context: HogQLContext) ->
     return response
 
 
-def property_access_to_clickhouse(chain: List[str]):
+def parse_field_access(chain: List[str], context: HogQLContext) -> HogQLFieldAccess:
     # Circular import otherwise
     from posthog.models.property.util import get_property_string_expr
 
     """Given a list like ['properties', '$browser'] or ['uuid'], translate to the correct ClickHouse expr."""
     if len(chain) == 2:
         if chain[0] == "properties":
+            if context.values is not None:
+                key = f"val_{len(context.values)}"
+                context.values[key] = chain[1]
+                escaped_key = f"%({key})s"
+            else:
+                escaped_key = escape_param(chain[1])
```

Comment:
should we be escaping this here?

Reply:
Looking back, this is actually is a remnant of an earlier time, before I added this value collection everywhere. The `format` strings just made it too risky to proceed. I left this in as an option, but there's no real reason to keep it. Removed.